kbase-extension/scss/partials/_kb-icon.scss

```diff
@@ -10,33 +15,52 @@ $icon-size-image: $icon-size-default - 2*$icon-padding;
     &--image {
       display: block;
     }
-  }
 
-  &__icon {}
+    /* kb-icon__container--data,
+      kb-icon__container--data-stack */
+    &--data,
+    &--data-stack {
+      cursor: pointer;
+    }
+
+    &--data,
+    &--app,
+    &--app-toolbar,
+    &--generic,
+    &--generic-toolbar,
+    &--type,
+    &--type-toolbar {
+      font-size: 2em;
+    }
+
+    &--data-stack {
+      font-size: 1.7em;
+    }
```

Comment:
Stacked data icons -- font size has to be adjusted downwards so they don't look too big.